be/src/storage/vectorized/column_eval_predicate.h

```diff
@@ -0,0 +1,155 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+
+#include "column/nullable_column.h"
+#include "storage/vectorized/column_predicate.h"
+
+namespace starrocks::vectorized {
+
+template <FieldType field_type, class ColumnType, template <FieldType> class PredicateOperator, typename... Args>
+class EvalPredicate final : public ColumnPredicate {
```

Comment:
Could this class name more descriptive? EvalPredicate is too much general. How about ColumnOperatorPredicate?

Reply:
Sounds great

Reply:
Done